cmd/sanitize-prow-jobs/main.go

```diff
@@ -15,27 +17,66 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-const defaultCluster = "api.ci"
-
 type options struct {
 	prowJobConfigDir string
+	configPath       string
 
 	help bool
 }
 
+type Config struct {
+	Clusters struct {
+		Default      string `json:"default"`
+		Distribution map[string]struct {
+			Jobs   []string `json:"jobs"`
+			Files  []string `json:"files"`
+			Groups []string `json:"groups"`
+		} `json:"distribution"`
```

Comment:
What about `assignment`? The distribution is more the result of the assignment we do here

Reply:
Done.